### PR TITLE
Rename redisDb to serverDb

### PR DESCRIPTION
**Pull request by 9bany**
A task of https://github.com/valkey-io/valkey/issues/144

### CODE REVIEW

**Comment by 9bany** [comment]
@zuiderkwast  check it out

File: Dockerfile
Comment (enjoy-binbin) [inline Dockerfile:1]: why we have this dockerfile?

Comment (9bany) [reply Dockerfile:1]: Sorry, my bad @enjoy-binbin

**Review by enjoy-binbin — APPROVED** [review]

**Comment by zuiderkwast** [comment]
@0del @enjoy-binbin Good job. This was during night time in my time zone. :)

**Pull request merged**